.pre-commit-config.yaml

```diff
@@ -16,14 +16,14 @@ repos:
     rev: 24.8.0
     hooks:
     -   id: black-jupyter
-        args: ['--line-length', '79']
+        args: ['--line-length', '50']
         language_version: python3
 -   repo: https://github.com/PyCQA/isort
     rev: 5.13.2
     hooks:
     -   id: isort
         args: ['--profile', 'black',
-               '--line-length', '79']
+               '--line-length', '50']
```

Comment:
Why this change? If anything, I have seen support around CFA for longer line lengths.